### ISSUE TITLE
Miscellaneous typography variables

### ISSUE THREAD

**Issue by juanca** [mavenlink/design-system#30]
We might want to consider evaluating the purposes of our defined variables. We might have jumped the gun and ported too many.

For example:

```
  --mavenlink-type-button-font-size: 14px;
  --mavenlink-type-body-font-size: 12px;
  --mavenlink-type-text-rendering: optimizeLegibility;
```

The button variable is better represented in the buttons style sheet.
The body font size and text rendering values seems too ambiguous in terms of our design system. Perhaps this is an indicator that we need to come up with a guide on structuring a new page to look like Mavenlink.

**Comment by roblevintennis** [comment]
Yes, these are very old values. I think Huth may have experimented with an app-wide increase in font size and it just broke too many things? In any event, I've heard from design folks that we definitely want to go larger, and 12px is obviously quite small. Perhaps we will need to purposely diverge for an interim period in this case (haha, opposite side of argument from yesterday! 😋) Let's plan to discuss what this all means.
